src/components/ui/deployments/DeleteDeployment.js

```diff
@@ -16,4 +16,10 @@ exports.clickCancel = function() {
     element.all(by.css('#deleteDialog .confirmationButtons button')).then(function(btns) {
         btns[0].click();
     });
+};
+
+exports.clickClose = function() {
```

Comment:
why not `$('.ngdialog-close').click()` or `$$('.ngdialog-close').first().click()`?